ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -719,6 +725,19 @@ open class NavigationBar: UINavigationBar {
         }
     }
 
+    private func updateSubtitleView(for navigationItem: UINavigationItem?) {
+        guard let navigationItem = navigationItem, let subtitle = navigationItem.subtitle else {
+            // Use the default title view
+            navigationItem?.titleView = nil
```

Comment:
How does setting the titleView to nil here equate to using the default title view?

Reply:
Seconded - if there's a way to make the behavior explicit (setting a defaultView bool, assigning a view that is constructed as a default view, etc), please do so. Side effects are an anti-pattern.

Reply:
This is how Apple's documentation says you can specify a custom view for a navigation bar.
https://developer.apple.com/documentation/uikit/uinavigationitem/1624935-titleview